### PR TITLE
DOC: Improved docs for jnp.fft.ifftn

### PR DESCRIPTION
**Pull request by rajasekharporeddy**
Part of #21461.

### CODE REVIEW

File: jax/_src/numpy/fft.py [hunk 1]
```diff
@@ -183,10 +183,71 @@ def fftn(a: ArrayLike, s: Shape | None = None,
...
+    a: input array
+    s: sequence of integers. Specifies the shape of the result. If not specified,
+      it will default to the shape of ``a`` along the specified ``axes``.
+    axes: sequence of integers, default=None. Specifies the axes along which the
+      transform is computed.
+    norm: string. The normalization mode. "backward", "ortho" and "forward" are
+      supported.
+
+  Returns:
+    An array containing the discrete fourier transform of ``a``.
```
Comment (jakevdp) [inline jax/_src/numpy/fft.py:204]:
```suggestion
    An array containing the inverse discrete Fourier transform of ``a``.
```

File: jax/_src/numpy/fft.py [hunk 2]
```diff
@@ -183,10 +183,71 @@ def fftn(a: ArrayLike, s: Shape | None = None,
...
+  given axes.
+
+  JAX implementation of :func:`numpy.fft.ifftn`.
+
+  Args:
+    a: input array
+    s: sequence of integers. Specifies the shape of the result. If not specified,
+      it will default to the shape of ``a`` along the specified ``axes``.
+    axes: sequence of integers, default=None. Specifies the axes along which the
+      transform is computed.
```
Comment (jakevdp) [inline jax/_src/numpy/fft.py:199]: We should mention the meaning of None here rather than in the example description.

**Review by jakevdp — CHANGES_REQUESTED** [review]

File: jax/_src/numpy/fft.py [hunk 3]
```diff
@@ -183,10 +183,71 @@ def fftn(a: ArrayLike, s: Shape | None = None,
   return _fft_core('fftn', xla_client.FftType.FFT, a, s, axes, norm)
 
 
-@implements(np.fft.ifftn)
 def ifftn(a: ArrayLike, s: Shape | None = None,
           axes: Sequence[int] | None = None,
           norm: str | None = None) -> Array:
+  r"""Compute inverse discrete Fourier transform of multidimensional array along
+  given axes.
```
[the remark this answers is not in the document]
Comment (rajasekharporeddy) [reply jax/_src/numpy/fft.py]: Removed

File: jax/_src/numpy/fft.py (hunk 2 above)
Comment (rajasekharporeddy) [reply jax/_src/numpy/fft.py]: Mentioned the meaning of None at the description of axes.

**Comment by rajasekharporeddy** [comment]
Modified the docstring and squashed the commits into one.

**Review by jakevdp — APPROVED** [review]
Thanks!

**Pull request merged**